README.md

```diff
@@ -14,7 +14,7 @@ Following actions are inside:
 
 ``LOOKUP BY PRIMARY KEY`` - this action will execute select query from specified table, as criteria can be used only [PRIMARY KEY](https://en.wikipedia.org/wiki/Primary_key "PRIMARY KEY"). The action returns only one result (a primary key is unique). 
 
-``UPSERT BY PRIMARY KEY`` - this action will execute select query from specified table, as criteria can be used only [PRIMARY KEY](https://en.wikipedia.org/wiki/Primary_key "PRIMARY KEY"), if result is nullable, then action will execute select query by [PRIMARY KEY](https://en.wikipedia.org/wiki/Primary_key "PRIMARY KEY"), else - action will execute update query by [PRIMARY KEY](https://en.wikipedia.org/wiki/Primary_key "PRIMARY KEY"). The action returns only one result (a primary key is unique).
+``UPSERT BY PRIMARY KEY`` - this action will execute select query from specified table, as criteria can be used only [PRIMARY KEY](https://en.wikipedia.org/wiki/Primary_key "PRIMARY KEY"), if result is nullable, then action will execute insert query by [PRIMARY KEY](https://en.wikipedia.org/wiki/Primary_key "PRIMARY KEY"), else - action will execute update query by [PRIMARY KEY](https://en.wikipedia.org/wiki/Primary_key "PRIMARY KEY"). The action returns only one result row (a primary key is unique).
```

Comment:
There were changes in query builders, and now the action will execute `Merge` SQL-statement instead of pair lookup-upsert. This behaviour was implemented for preventing conflicts in the process with other sessions.